### PR TITLE
Update change-the-phone-numbers-on-your-audio-conferencing-bridge.md

### PR DESCRIPTION
**Pull request by pupara**
Added Notes
Replaced SkypeOnlineConnector with MicrosoftTeams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md [hunk 1]
```diff
@@ -166,9 +166,13 @@ If you need to know more, see [Connect to all Microsoft 365 or Office 365 servic
 
 2. In the **Windows PowerShell** window, connect to Microsoft 365 or Office 365 by running:
 
+> [!NOTE]
+> Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md:170]:
```suggestion
> The Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**